README.md

```diff
@@ -173,6 +175,35 @@ Use Google doc to enable in-line comments and suggestions.
 
 *Allocate enough time for your reading, discuss as a group, write the summary carefully, and finally, include key observations from the class discussion.*
 
+### Post-Presentation Panel Discussion 
+To foster a deeper understanding of the papers and encourage critical thinking, each lecture will be followed by a panel discussion. 
+This discussion will involve three distinct roles played by different student groups, simulating an interactive and dynamic scholarly exchange.
+Roles and Responsibilities:
+
+1. The Authors:
+- Group Assignment: The group that presents the paper will play the role of the paper's authors.
+- Responsibility: As authors, you are expected to defend your paper against critiques, answer questions, and discuss how you might improve or extend your research in the future, akin to writing a rebuttal during the peer-review process.
+
+
+2. The Reviewers:
+- Group Assignment: Each group will **sign up for one slot to play the role of reviewers**.
```

Comment:
not sign up for one slot; rather assigned. this probably shouldn't be the same groups that are presenting or summarizing but a different group.

Reply:
make sense.

Reply:
> Also, we can assign two groups to be reviewers instead of one.
having 12 people participate in the panel discussion seems reduce the opportunity of speaking up for each student, given the limited time frame. Too many people do not lead to depth discussion, but fragmented discussion. 
Also, the 'reviewers' basically only need to complain, but we want the authors to speak more. With the current plan, each team will still need to join panel >= 3 times.
What do you think?